test/fixtures/typescript-order-custom-paths-mapping/tsconfig-with-path-mapping.json

```diff
@@ -0,0 +1,15 @@
+{
+  "compilerOptions": {
+    "paths": {
+      "@/*": [
+        "./src/*"
+      ],
+      "@components/*": [
+        "./src/components/*"
+      ],
+      "#private": [
+        "./src/private/*"
```

Comment:
Sorry but are there real files for testing them? It could affect the result w/ resolved or not.

Reply:
Could you specify what you mean? There should be no files, because the import/order rule just orders the imports, same as ts import via LSP. This rule or anything else only knows the content of the file itself, and should not know if there are other real, virtual or whatever files exists when ordering the imports.

Reply:
Whether resolved would affect the `isExternalPath` logic.

https://github.com/un-ts/eslint-plugin-import-x/blob/9769f3cb30e99f98fb67a05682b74164d1a87f41/src/utils/import-type.ts#L109-L111

Reply:
Ok, can add some files, but I'm away now for some hours 
Soon™️

Reply:
@JounQin added now the files in [`e4ff449` (#287)](https://github.com/un-ts/eslint-plugin-import-x/pull/287/commits/e4ff4497a9e1fe8fc983ac0bbf4633d6fb96f1e1)
Let me know if I did it wrong (because tbh it confuses me a bit why real files are necessary to order imports in another file, I mean even on a concept level 🤷)

Reply:
> because tbh it confuses me a bit why real files are necessary to order imports in another file, I mean even on a concept level 🤷

That's how `isExternalPath` works? If there is no such **real** file, then it could be a virtual file which should not be considered as external like https://rspress.dev/plugin/system/plugin-api#addruntimemodules.

Reply:
There are two parts:
1. the "I" don't care, and I like to satisfy you wishes 😇
2. the "I" don't understand what's really going on and why it needs to be so complex... but 1.: "I don't care" 😆 

and yeah, I understand maintenance burdens as I'm also maintainer of many taken over Open Source projects